services/order.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from db.models import Ticket, Order, User
+
+
+def create_order(tickets: list,
+                 username: str,
+                 date: str = None) -> Order:
+    with transaction.atomic():
+        # Перевірка на існування користувача
+        try:
+            user = get_user_model().objects.get(username=username)
+        except User.DoesNotExist:
+            raise ValueError("User does not exist")
```

Comment:
Raising a `ValueError` here is a good approach to handle the case where a user does not exist. However, consider using a more specific exception type or logging the error for better error management and debugging.